x/pairing/genesis.go

```diff
@@ -37,9 +43,48 @@ func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
 	genesis := types.DefaultGenesis()
 	genesis.Params = k.GetParams(ctx)
 
-	genesis.UniquePaymentStorageClientProviderList = k.GetAllUniquePaymentStorageClientProvider(ctx)
-	genesis.ProviderPaymentStorageList = k.GetAllProviderPaymentStorage(ctx)
-	genesis.EpochPaymentsList = k.GetAllEpochPayments(ctx)
+	epochs, uniqueEpochSessions := k.GetAllUniqueEpochSessionStore(ctx)
+	for i := range epochs {
+		ues := types.UniqueEpochSessionGenesis{
+			Epoch:              epochs[i],
+			UniqueEpochSession: uniqueEpochSessions[i],
+		}
+		genesis.UniqueEpochSessions = append(genesis.UniqueEpochSessions, ues)
+	}
+
+	epochs, keys, providerEpochCus := k.GetAllProviderEpochCuStore(ctx)
+	for i := range epochs {
+		provider, chainID, err := types.DecodeProviderEpochCuKey(keys[i])
+		if err != nil {
+			utils.LavaFormatError("could not decode ProviderEpochCu key", err, utils.LogAttr("key", keys[i]))
+			continue
+		}
+		pec := types.ProviderEpochCuGenesis{
+			Epoch:           epochs[i],
+			Provider:        provider,
+			ChainId:         chainID,
+			ProviderEpochCu: providerEpochCus[i],
+		}
+		genesis.ProviderEpochCus = append(genesis.ProviderEpochCus, pec)
+	}
+
+	epochs, keys, providerConsumerEpochCus := k.GetAllProviderConsumerEpochCuStore(ctx)
```

Comment:
and here

Reply:
done. see https://github.com/lavanet/lava/commit/ebb7dc05fbcd023b4d71351daf090150b9322ca1